test/AutoDiff/compiler_crashers/tf881-derivative-local-variable-capture.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: not --crash %target-swift-emit-silgen %s
+// RUN: %target-swift-emit-silgen %s -verify
```

Comment:
Please move this test from `test/AutoDiff/compiler_crashers` to `test/AutoDiff/compiler_crashers_fixed`.